data/kramdown-rfc2629.erb

```diff
@@ -47,17 +47,29 @@
 <%  if venue -%>
 <%    venue = KramdownRFC::ParameterSet.new(venue) -%>
     <note title="Discussion Venues" removeInRFC="true">
-<%   if mail = venue[:mail] -%>
+<%   mail = venue[:mail] -%>
+<%   homepage = venue[:homepage] -%>
+<%   gtype = venue[:type] -%>
```

Comment:
Why is this up here (trying to understand the logic)

Reply:
gtype is used in the "homepage" block as well; we can't infer the type from the mailing list if there is no mailing list, but you can specify "type" and "homepage" and get a sensible output.